### PR TITLE
pharnb update

### PR DESCRIPTION
**Pull request by pharnb**
Non-image, resume, more info

### CODE REVIEW

**Comment by pharnb** [comment]
@rtrp/admins

File: users/pharnb/index.md [hunk 1]
```diff
@@ -6,10 +6,30 @@ author: Ben Pharn
...
 
-I work under Todd Palmer, and I am currently working on a project to help develop safeguards for small modular reactors. 
\ No newline at end of file
+I work under Todd Palmer, and I am currently working on a project to help develop safeguards for small modular reactors for Idaho National Laboratory.
+
+## Affiliations
+
+* American Nuclear Society: September 2012 - Present
+  * Member
+  * Local Chapter Webmaster September 2015 - June 2016
```
Comment (albeanth) [inline users/pharnb/index.md]: this is me being nit picky.... OSU has an ANS student [section](http://students.ans.org/) not a chapter.

**Comment by dougwoods3** [comment]
:shipit:

**Comment by harterj** [comment]
@albeanth semantics... :shipit:

File: users/pharnb/index.md (hunk 1 above)
Comment (pharnb) [reply users/pharnb/index.md]: No problem, better to be accurate on something like this.  Thanks!

**Comment by harterj** [comment]
@pharnb 👍

**Comment by harterj** [comment]
@albeanth I think you did!

**Pull request merged**